src/Manager/DesordrePrecisionManager.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Manager;
+
+use App\Entity\DesordrePrecision;
+use App\Entity\Enum\Qualification;
+use Doctrine\Persistence\ManagerRegistry;
+
+class DesordrePrecisionManager extends AbstractManager
+{
+    public function __construct(
+        protected ManagerRegistry $managerRegistry,
+        string $entityName = DesordrePrecision::class
+    ) {
+        parent::__construct($managerRegistry, $entityName);
+    }
+
+    public function createOrUpdate(string $slug, array $data): DesordrePrecision
+    {
+        /** @var DesordrePrecision|null $desordrePrecision */
+        $desordrePrecision = $this->getRepository()->findOneBy([
+            'desordrePrecisionSlug' => $slug,
+        ]);
+        if (null === $desordrePrecision) {
+            $desordrePrecision = (new DesordrePrecision());
+        }
+
+        $coef = str_replace(',', '.', $data['coef']);
```

Comment:
Vérifier que $data['coef'] est défini pour éviter les messages d'avertissement sur les fonction string php
`PHP Deprecated:  str_replace(): Passing null to parameter ... `